src/core/Target.mli

```diff
@@ -0,0 +1,146 @@
+(* Cooper Pierce
+ *
+ * Copyright (c) Semgrep Inc.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation, with the
+ * special exception on linking described in file LICENSE.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the file
+ * LICENSE for more details.
+ *)
+
+(** Types for describing targets.
+
+    See also {!Input_to_core_t}, which has a similar set of types used when
+    pysemgrep generates targets that have slightly less information (e.g.,
+    these types have expanded information about the targets' locations). *)
+
+(** The location of a Semgrep target. This contains all of the target-specific
```

Comment:
Seems like it's actually not just the location of a Semgrep target, so I would remove this leading comment.

Reply:
Ah yes, this is just an artifact of the fact that this was previously `Target_location`. I'll revise to make sure this is consistent with the Target/Xtarget naming.

Reply:
Fixed in f8c7725030537be263c25510c3a0cc5f92a5d510.